### ISSUE TITLE
Would this work on a C++ codebase?

### ISSUE THREAD

**Issue by olilarkin** [prolego-team/groundcrew#33]
Thanks for the nice example.

I started running it on my C++ codebase having adapted the file extensions in config.yml, but then I realized that it looks like some of the code is designed for looking at python. Is that true? Is this limited to python code bases in its current form?

**Comment by jmpounders** [comment]
Yes, the current implementation is focused on Python and has some Python-specific tools. Some of the tools may work for other languages, but it was not designed with other languages in mind.

The conceptual framework used here would apply to any language, but the tooling used by the agent would need to be changed.
